Dockerfile

```diff
@@ -11,7 +11,7 @@ WORKDIR /ui
 
 RUN npm install
 
-RUN gatsby build
+RUN PATH_PREFIX='/' gatsby build --prefix-paths
```

Comment:
as per docs, it looks like `PATH_PREFIX` is a boolean and not for passing a path - https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/path-prefix/

also, it says that we either pass `PATH_PREFIX` or the `--prefix-paths` flag.

Reply:
sorry I got mixed with the name. I also mentioned in the UI PR that it would be clearer to name the variable `KEPLOY_PATH_PREFIX `.

Reply:
I referred this doc for setting PATH_PREFIX https://github.com/gatsbyjs/gatsby/issues/7899 to build using gatsby

Reply:
will update variable name

Reply:
Done with changes Thanks!!

Reply:
`PATH_PREFIX` in that issue was just a random variable. Thanks for updating.

Reply:
also, it would be good to pass a build arg in the dockerfile. eg: https://stackoverflow.com/questions/66561671/pass-arguments-in-dockerfile

Reply:
Updated with Arg changes. @slayerjain  could you please review it?